perfect_share.py

```diff
@@ -0,0 +1,35 @@
+def is_prime(num):
+    if num > 1:
+        for n in range(2, num):
```

Comment:
Do we must check all of the numbers until num? 
(Is there a divisior that is bigger than num // 2?)
Also, dont use single letter names (n) and give num a better name.